app-shells/zsh/zsh-5.3.1.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A UNIX command interpreter (shell)"
+DESCRIPTION="Zsh is a shell designed for interactive use, although it \
+is also a powerful scripting language. Many of the useful features of \
+bash, ksh, and tcsh were incorporated into zsh; many original features \
+were added."
+HOMEPAGE="https://zsh.org/"
+COPYRIGHT="1992-2017 Paul Falstad, Richard Coleman, Zoltán Hidvégi, \
+Andrew Main, Peter Stephenson, Sven Wishnowsky, and others"
+LICENSE="ZSH"
+REVISION="1"
+SOURCE_URI="https://downloads.sf.net/zsh/zsh-$portVersion.tar.xz"
+CHECKSUM_SHA256="fc886cb2ade032d006da8322c09a7e92b2309177811428b121192d44832920da"
+PATCHES="zsh-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	zsh = $portVersion compat >= 5
+	cmd:zsh = $portVersion compat >= 5
+	cmd:zsh_$portVersion = $portVersion compat >= 5
+	#zsh isn't bash compatible enough
+	#cmd:zsh_$portVersion = $portVersion compat >= 5
+	"
+REQUIRES="
+	haiku
+	lib:libiconv
+	lib:libncursesw
+	lib:libpcre
+	lib:libiconv
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libiconv
+	devel:libncursesw
+	devel:libpcre
+	devel:libiconv
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:awk
+	cmd:find
+	cmd:gcc
+	cmd:grep
+	cmd:libtoolize
+	cmd:ld
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
+{
+	autoreconf -fi
+	libtoolize -vfi
+
+	export LDFLAGS="-lnetwork -lbsd"
+	export CFLAGS="-D_BSD_SOURCE"
+
+	runConfigure ./configure \
+		--enable-multibyte \
+		--enable-zshrc=`finddir B_USER_SETTINGS_DIRECTORY`/zsh/zshrc \
+		--sysconfdir=`finddir B_USER_SETTINGS_DIRECTORY`/zsh \
+		--enable-etcdir=`finddir B_USER_SETTINGS_DIRECTORY`/zsh \
+		--enable-pcre
+	sed --in-place '/HAVE_WAIT3/d' config.h
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	#zsh isn't bash compatible enough
+	#ln -s zsh $binDir/sh
```

Comment:
If zsh  enabled on my machine with cmd:sh it generates serious problems during configuring thngs with automake (autmake scripts using /bin/sh almost all the times). It is a bug, it shouldn't let break anything.
Without zsh the works correctly. We should let /bin/sh for bash right now.